backend/store/etcd/generic_object.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/sensu/sensu-go/api/core/v2/meta.proto";
+
+package sensu.etcd;
+
+option go_package = "etcd";
+option (gogoproto.populate_all) = true;
+option (gogoproto.equal_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.testgen_all) = true;
+
+// Asset defines an asset agents install as a dependency for a check.
+message GenericObject {
```

Comment:
FYI we use this resource to test the generic store so obviously, to have it working with protobuf I had to create this file.